### PR TITLE
ci: Split one of the system test child jobs to reduce run time.

### PR DESCRIPTION
**Pull request by jgsbennett**

### CODE REVIEW

**Comment by jgsbennett** [comment]
Testing here:
https://jenkins.ops.vega.xyz/job/common/job/system-tests/18284/

**Comment by jgsbennett** [comment]
Retry after fixing my bug:
https://jenkins.ops.vega.xyz/job/common/job/system-tests/18367/
(Was specifying the [pP]* as additional pytest args, rather than overriding the directory, so the child job was running _both_ the tests I wanted _and_ all network_infra tests.

File: vars/pipelineCapsuleSystemTests.groovy [hunk 1]
```diff
@@ -48,13 +48,18 @@ void call() {
...
-      'network_infra a-n': [
-        pytestDirectory: "tests/[a-nA-N]*",
+      'network_infra a-o': [
+        pytestDirectory: "tests/[a-oA-O]*",
         mark: 'network_infra',
         capsuleConfig: 'capsule_config_network_infra.hcl'
       ],
-      'network_infra o-z without validators': [
-        pytestArgs: "--ignore-glob 'tests/[a-nA-N]*' --ignore-glob 'tests/validators/*'",
+      'network_infra p without validators': [
```
Comment (daniel1302) [inline vars/pipelineCapsuleSystemTests.groovy:56]: What with tests `N`?

Comment (jgsbennett) [reply vars/pipelineCapsuleSystemTests.groovy:56]: I believe n tests would go in the a-o job. "...l, m, n , o..."

**Review by daniel1302 — APPROVED** [review]

**Comment by jgsbennett** [comment]
https://jenkins.ops.vega.xyz/job/common/job/system-tests/18284/rebuild/parameterized

**Pull request merged**